pkg/daemon/ceph/multus/setup.go

```diff
@@ -0,0 +1,81 @@
+package multus
+
+import (
+	"errors"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+)
+
+func Setup() error {
+
+	logger.Info("Checking if interface has already been migrated")
```

Comment:
no capital letters in log messages

Reply:
What's the reasoning for this? Is it a convention?

Reply:
Yes, it's Rook-Ceph's convention, just like error messages which are more of a Golang convention. Rook-Ceph does not do this.